pegasus/sites.v3/code.org/public/csd/standards.moved

```diff
@@ -1 +0,0 @@
-/educate/csd/standards
```

Comment:
please reinstate this one, because we don't have a good way to tell if it is being used (GA doesn't log anything for redirects like this)

Reply:
@davidsbailey done!